conan/api/subapi/upload.py

```diff
@@ -99,6 +187,9 @@ def _upload_pkglist(pkglist, subtitle=lambda _: None):
             thread_pool.join()
         elapsed = time.time() - t
         ConanOutput().success(f"Upload completed in {int(elapsed)}s\n")
+        if dry_run:
```

Comment:
Maybe this is not to be called dry-run, maybe even it is not necessary to be optional, it can always be added unconditionally to the package list the URLs? 
Like it won't cost extra time, it is very cheap, it doesn't break to add extra information to the json of the package list...